mainflux/boostrap.py

```diff
@@ -0,0 +1,143 @@
+import requests
+
+from mainflux import response
+from mainflux import errors
+from mainflux import utils
+
+
+class Bootstrap:
+    configs_endpoint = "configs"
```

Comment:
Uppercase letters for constants, i think that is practise in Python, 
@drasko what do you say ?

Reply:
@fbugarski correct, this is idiomatic way of writing constants in Python, and is documented in their code style PEP: https://peps.python.org/pep-0008/#constants